runtime/compiler/env/SystemSegmentProvider.cpp

```diff
@@ -22,13 +22,25 @@
 
 #include "SystemSegmentProvider.hpp"
 #include "env/MemorySegment.hpp"
+#include "control/J9Options.hpp"
+#include <stdio.h>
 #include <algorithm>
+#include <vector>
+#include <tuple>
+
+uint32_t J9::SystemSegmentProvider::_globalCompilationSequenceNumber = 0;
+std::vector<std::tuple<uint32_t, size_t, RegionLog *>> *J9::SystemSegmentProvider::_globalCompilationsList = new std::vector<std::tuple<uint32_t, size_t, RegionLog *>>;
```

Comment:
Why do these need to use `malloc` rather than the persistent allocator?

Reply:
Oh, yes. How to access the persistent allocator here?

Reply:
Same as anywhere else, e.g. `TR::Compiler->persistentAllocator()`.